Automattic/jetpack
# Stats widget: minor cleanup of Akismet section

Cleaning up a couple of things I noticed when using the Stats widget with Akismet for the first time.

## Proposed changes:
1. Add some breathing space to the action buttons, lining up with the adjacent count if present.

Before: 

<img width="503" alt="Screenshot 2023-03-08 at 10 59 08 AM" src="https://user-images.githubusercontent.com/17325/223569085-1f1d2945-0171-41c3-9e4b-a2d8830ac714.png">

After: 
<img width="517" alt="Screenshot 2023-03-08 at 10 56 00 AM" src="https://user-images.githubusercontent.com/17325/223569152-fb0df7ab-0529-4c2b-93dc-6588d8508537.png">
<img width="514" alt="Screenshot 2023-03-08 at 10 59 00 AM" src="https://user-images.githubusercontent.com/17325/223569162-5781c32b-c4fc-44f8-84f1-2d2db2927fea.png">

2. Make full stop use consistent. The Protect count didn't have one, but the Akismet count did. I've removed the Akismet one.

Before:

<img width="505" alt="Screenshot 2023-03-08 at 11 20 35 AM" src="https://user-images.githubusercontent.com/17325/223569322-4fe615b8-c274-4e79-b659-99f517dafa54.png">

After: 

<img width="503" alt="Screenshot 2023-03-08 at 11 35 26 AM" src="https://user-images.githubusercontent.com/17325/223569403-71b6114e-0543-4461-825a-896d6efcaad4.png">

### Other information:


## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
1. Apply the PR and ensure the styles are rebuilt (`pnpm jetpack watch plugins/jetpack` will do the trick).
2. Visit the wp-admin dashboard `/wp-admin/index.php` and locate the Jetpack Stats widget.
